### ISSUE TITLE
design level : .png VS .json

### ISSUE THREAD

**Issue by jamet-julien** [meth-meth-method/super-mario#117]
use the pixels of a .png image to build the design of the levels. for example a red pixel for a brick and a blue pixel for a turtle. With image analysis rather than level json

**Comment by bouveng** [comment]
Where should we get this .png from? Create it ourselves? Placing pixels in an image might not be faster than writing the x, y in a json.

A level editor would be nice, but building a level editor might take as much time as just creating the levels in json.

😎

**Comment by jamet-julien** [comment]
indeed it could be an image made in photoshop for example. See with Paint. Unlike JSON is that we don't need to know how to coordinate them but just make a drawing with a color by type of element. This makes the construction intuitive on a drawing software and quickly correctable without having to look in the JSON for the right brick to modify. A level editor would be too expensive in time but it is true that it would be even better ... but here I just suggest a parse of image.

**Comment by kktos** [comment]
> A level editor would be nice, but building a level editor might take as much time as just creating the levels in json.

hum... we may think about it as the game renderer with no move (vel=0) and the camera following the mouse moves. A mouse click could display a square around the targeted tile and a choice of keyboard keys could allow us to do actions....
What do you think about it ?

**Comment by jamet-julien** [comment]
> > A level editor would be nice, but building a level editor might take as much time as just creating the levels in json.
> 
> hum... we may think about it as the game renderer with no move (vel=0) and the camera following the mouse moves. A mouse click could display a square around the targeted tile and a choice of keyboard keys could allow us to do actions....
> What do you think about it ?

yes it's a good idea and not incompatible with the recording mode image 😉
